__tests__/app.test.js

```diff
@@ -103,8 +103,45 @@ describe("GET /api/articles/:article_id", ()=>{
     })
     test("404: responds with an error message when passed a valid, but non existent article ID", ()=>{
         return request(app).get("/api/articles/123123").expect(404).then((response)=>{
-            expect(response.body).toEqual({message: "No article found"})
+            expect(response.body).toEqual({message: "404 Not Found"})
         })
     })
+})
 
+describe("GET /api/articles", ()=>{
+    test("200: responds with a status of 200", ()=>{
+        return request(app).get("/api/articles").expect(200).then((response)=>{
+            expect(response.status).toBe(200)
```

Comment:
the expect(200) is enough to check the status code you are recieving, this also doesn't need to be a seperate test and you can include .expect(200) in your next test.